src/main/java/com/n1303/birthdaybuddy/repository/FriendshipRepository.java

```diff
@@ -0,0 +1,31 @@
+package com.n1303.birthdaybuddy.repository;
+
+import com.n1303.birthdaybuddy.entity.Friendship;
+import org.springframework.stereotype.Repository;
+
+import java.util.*;
+
+@Repository
+public class FriendshipRepository {
+
+    private final Map<String, Friendship> friendshipMap;
+
+    public FriendshipRepository() {
+        this.friendshipMap = new HashMap<>();
+    }
+
+    public List<Friendship> getAll() {
+        return new ArrayList<Friendship>(this.friendshipMap.values());
+    }
+
+    public void insert(String firstUser, String secondUser) {
```

Comment:
Although it's a test project, It'll be better if we add existence check here before inserting

Reply:
Sure.